### PR TITLE
Fixed red planet color

### LINKED ISSUE #1269: sojourner.wbt doesn't look like Mars any more
Because of recent changes in the textures / background, the ground color of sojourner.wbt is not red any more, but grey. It's a pity as it should refer to the red planet (Mars)...

### PR DESCRIPTION
**Pull request by omichel**
Fix #1269.
Before this PR:
sojourner_before
After this PR:
sojourner_new

### CODE REVIEW

**Comment by omichel** [comment]
👍

File: projects/objects/rocks/protos/Rock10cm.proto [hunk 1]
```diff
@@ -8,6 +8,7 @@ PROTO Rock10cm [
   field SFRotation rotation         0 1 0 0
   field SFString   name             "rock 10 cm"
   field MFString   texture          "textures/rock.jpg"  # Defines the texture used for the rock.
+  field SFColor    color            1 1 1                # Is `Solid.chidlren.Shape.appearance.baseColor`.
```
Comment (DavidMansolino) [inline projects/objects/rocks/protos/Rock10cm.proto]: This comment does not follow our PROTO guidelines:
> If the field is connected directly to a **root** node with an IS, the comment should be # Is RootNodeType.fieldname.

**Comment by omichel** [comment]
👍

File: projects/objects/rocks/protos/Rock10cm.proto [hunk 2]
```diff
@@ -8,6 +8,7 @@ PROTO Rock10cm [
   field SFRotation rotation         0 1 0 0
   field SFString   name             "rock 10 cm"
   field MFString   texture          "textures/rock.jpg"  # Defines the texture used for the rock.
+  field SFColor    color            1 1 1                # Defines the base color of the rock
```
Comment (stefaniapedrazzi) [inline projects/objects/rocks/protos/Rock10cm.proto]: Just a minor note: here the final `.` is missing in the field description

Comment (DavidMansolino) [reply projects/objects/rocks/protos/Rock10cm.proto]: I will add a test for this in our test suite :wink:

**Comment by DavidMansolino** [comment]
This planet is actually not so red:
Capture du 2020-07-27 08-31-17
Left: the first image published by the NASA that made people think the planet is red. Right: actual color of the planet.
> https://www.numerama.com/sciences/638963-mars-la-planete-pas-si-rouge-que-ca.html

**Pull request merged**